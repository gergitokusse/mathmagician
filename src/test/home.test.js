import { render, screen } from '@testing-library/react';
import Home from '../component/home';

describe('Test Home Component', () => {
  test('Text content testing', () => {
    render(<Home />);
    const pElement = screen.getByText(/Welcome to Our Page/i);
    expect(pElement).toMatchSnapshot();
  });
});
