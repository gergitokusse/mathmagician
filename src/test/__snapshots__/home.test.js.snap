// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home Component Text content testing 1`] = `
<h1>
  Welcome to Our Page
</h1>
`;
